packages/openapi-generator/src/file-serializer/docs.ts

```diff
@@ -0,0 +1,71 @@
+import { EOL } from 'os';
+import { documentationBlock } from '@sap-cloud-sdk/util';
+import {
+  OpenApiApi,
+  OpenApiDocument,
+  OpenApiOperation,
+  OpenApiParameter,
+  OpenApiRequestBody
+} from '../openapi-types';
+import { serializeSchema } from './schema';
+
+export function apiDocumentation(
+  api: OpenApiApi,
+  document: OpenApiDocument
+): string {
+  return documentationBlock`
+  Representation of the ${api.name} API.
+  This API is part of the ${document.serviceName} service.
+  
+  This API client has been created automatically using the SAP Cloud SDK - do not edit manually.
```

Comment:
This is very useful, but consider putting it on top of the file. I think we already have something along with the copyright.